packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "datacenter_backup_legacy_message_1": "Depuis le 25/02/2020, la gamme Veeam Backup a évolué.",
+  "datacenter_backup_legacy_message_2": "Vous pouvez choisir entre 3 nouvelles offres pour protéger vos machines virtuelles (VM) : <strong>Standard</strong>, <strong>Advanced</strong> ou <strong>Premium</strong>.",
```

Comment:
```suggestion
  "datacenter_backup_legacy_message_2": "Vous pouvez choisir entre 3 nouvelles offres pour protéger vos machines virtuelles (VM) : <strong>Standard</strong>, <strong>Advanced</strong> ou <strong>Premium</strong>.",
```
  
  non-breaking space

Reply:
@FredericEspiau  what do you mean by non-breaking space, both look the same for me.